code/modules/projectiles/projectile/special.dm

```diff
@@ -69,6 +69,7 @@
 	desc = "Boom"
 	icon_state = "missile"
 	damage = 30
+	ricochets_max = 0 //it's a MISSILE
```

Comment:
@Cyberboss 

That's for when something that returns `isReflect`, not for wall ricochets.